Source/src/components/ComponentTransform.cpp

```diff
@@ -47,14 +47,14 @@ inline Quat Hachiko::ComponentTransform::SimulateLookAt(const float3& direction)
 
     float3 up_temp = direction.Cross(right_temp).Normalized();
 
-    Quat orientation = Quat::LookAt(-float3::unitZ, direction, float3::unitY, float3::unitY);
+    Quat orientation = Quat::LookAt(float3::unitZ, direction, float3::unitY, float3::unitY);
```

Comment:
This is the look at fix, this does not affect engine since all engine functionality calles LookAtTarget which has been corrected to send the correct direction